include/behaviortree_cpp_v3/behavior_tree.h

```diff
@@ -56,10 +57,15 @@ void applyRecursiveVisitor(const TreeNode* root_node,
 void applyRecursiveVisitor(TreeNode* root_node, const std::function<void(TreeNode*)>& visitor);
 
 /**
- * Debug function to print on screen the hierarchy of the tree.
+ * Debug function to print on screen the hierarchy of the tree to std::cout.
  */
 void printTreeRecursively(const TreeNode* root_node);
 
+/**
+ * Debug function to print the hierarchy of the tree to a custom stream.
+ */
+void printTreeRecursively(const TreeNode* root_node, std::ostream& stream);
```

Comment:
can't we just have 

> void printTreeRecursively(const TreeNode* root_node, std::ostream& stream = std::cout );

For simplicity?

Reply:
Good idea, changed it to a default argument